src/h5web/visualizations/heatmap/models.ts

```diff
@@ -0,0 +1,83 @@
+import {
```

Comment:
I wonder if `models` is a good file name given it only exports color maps.

Reply:
Yeah, perhaps `interpolators.ts` would make more sense.